upgrade_guide/README.md

```diff
@@ -3,6 +3,19 @@
 The purpose of this guide is to detail changes made by successive
 versions of the Java driver.
 
+### 3.2.0
+
+[JAVA-1257](https://datastax-oss.atlassian.net/browse/JAVA-1257) has deprecated the `QueryLogger`
+class and users are now encouraged to use `EnhancedQueryLogger` instead, which makes it easier for
+users to customize the way statements are logged. 
+Please note that `QueryLogger` class might be removed in a future major release.
+
+[JAVA-1257](https://datastax-oss.atlassian.net/browse/JAVA-1257) also modifies the behavior 
```

Comment:
>I don't think we ever documented the fact that it returned the Query string 

Unfortunately for us, [we did it](https://github.com/datastax/java-driver/blob/3.1.x/driver-core/src/main/java/com/datastax/driver/core/RegularStatement.java#L192-L209) for `RegularStatement` :(